Makefile

```diff
@@ -91,12 +91,40 @@ port-forward-api:  ## port forward to service `teams-api` on the host port 8000
 port-forward-mongo:  ## port forward to service `mongodb` on the host port 27017
 	kubectl port-forward --namespace fiftyone-teams svc/mongodb 27017:27017 --context minikube
 
+tunnel:  ## run minikube tunnel to access the k8s ingress via localhost ()
+	minikube tunnel
+
```

Comment:
did you UNalphabetize this?

Reply:
Only because it seemed better to group the minkube-y things together than have them spread all over.

Reply:
If we prefer alpha sorting over logical grouping, I can update the rest of the file too.